CHANGELOG.md

```diff
@@ -6,8 +6,18 @@ This file was introduced starting Kani 0.23.0, so it only contains changes from
 
 ## [0.41.0]
 
+## What's Changed
+* Fix setup for `aarch64-unknown-linux-gnu` platform by @adpaco-aws in https://github.com/model-checking/kani/pull/2864
+* Do not override `std` library during playback by @celinval in https://github.com/model-checking/kani/pull/2852
+* Fix docker build step in release workflow by @adpaco-aws in https://github.com/model-checking/kani/pull/2854
+* Fix path-dependent failures in `assess-artifacts` test by @adpaco-aws in https://github.com/model-checking/kani/pull/2849
+* Automatic toolchain upgrade to nightly-2023-11-11 by @github-actions in https://github.com/model-checking/kani/pull/2870
```

Comment:
No idea if this OK because we're missing the information on the auto-generated release notes.